tableau-connector/src/dialect.tdd

```diff
@@ -598,6 +598,22 @@
       <argument type='str' />
       <argument type='int' />
     </function>
+    <function group='string' name='FIND' return-type='int'>
```

Comment:
Support passing float arguments to FIND() and other string functions as required by TDVT. Use X - X%1 to truncate the decimal portion. If FLOOR() is ever implemented can replace with that.

Reply:
I'm not sure MOD is an appropriate operator for real numbers. Can you add a test?

Reply:
Added test and added task to possibly implement floor and ceiling with this logic https://bitquill.atlassian.net/browse/AD-502